test/pact_test.exs

```diff
@@ -68,4 +55,28 @@ defmodule PactTest do
     Pact.put(:string, Integer)
     assert Pact.get("string") == Integer
   end
+
+  test "replace module replaces the given module" do
+    Pact.put("foo", String)
+
+    Pact.replace self, "foo" do
+      def awesome? do
+        true
+      end
```

Comment:
That seems like an implementation detail of the function. I think it would be good to make sure it can handle multiple functions. It will also serve as documentation so that people know you can define multiple functions.

Maybe make the `def` calls on line? `def awesome?, do: true`?